doc/papers/joss/paper.bib

```diff
@@ -0,0 +1,383 @@
+                   2.0}},
+  year         = 2019,
+  publisher    = {Zenodo},
+  doi          = {10.5281/zenodo.2641316}
+}
+
+@misc{CEEDMS34,
+  author       = {Kolev, Tzanio and
+                  Fischer, Paul and
+                  Abdelfattah, Ahmad and
+                  Ananthan, Shreyas and
+                  Barra, Valeria and
+                  Beams, Natalie and
+                  Bleile, Ryan and
+                  Brown, Jed and
+                  Carson, Robert and
+                  Camier, Jean-Sylvain and
+                  Churchfield, Matthew and
+                  Dobrev, Veselin and
+                  Dongarra, Jack and
+                  Dudouit, Yohann and
+                  Karakus, Ali and
+                  Kerkemeier, Stefan and
+                  Lan, YuHsiang and
+                  Medina, David and
+                  Merzari, Elia and
+                  Min, Misun and
+                  Parker, Scott and
+                  Ratnayaka, Thilina and
+                  Smith, Cameron and
+                  Sprague, Michael and
+                  Stitt, Thomas and
+                  Thompson, Jeremy and
+                  Tomboulides, Ananias and
+                  Tomov, Stanimire and
+                  Tomov, Vladimir and
+                  Vargas, Arturo and
+                  Warburton, Tim and
+                  Weiss, Kenneth},
+  title        = {{CEED ECP Milestone Report: Improve performance and
+                   capabilities of CEED-enabled ECP applications on
+                   Summit/Sierra}},
+  year         = 2020,
+  publisher    = {Zenodo},
+  doi          = {10.5281/zenodo.3860804}
+}
+
+@misc{CEEDwebsite,
+  title = "CEED",
+  url = "https://ceed.exascaleproject.org/",
+  year = "2020"
+}
+
+@misc{CUDAwebsite,
+  title = "CUDA",
+  url = "https://developer.nvidia.com/about-cuda",
+  year = "2020"
+}
+
+@article{Fischer2020scalability,
+  title={Scalability of high-performance PDE solvers},
+  author={Paul Fischer and Misun Min and Thilina Rathnayake and Som Dutta and Tzanio Kolev and Veselin Dobrev and Jean-Sylvain Camier and Martin Kronbichler and Tim Warburton and Kasia Świrydowicz and Jed Brown},
+  journal={The International Journal of High Performance Computing Applications},
+  year={2020},
+  doi = "10.1177/1094342020915762"
+}
+
+@book{Fortran77-lang,
+  title={Standard X3. 9-1978, Programming Language Fortran (revision of ANSI X2. 9-1966)},
+  author={ANSI},
+  journal={New York: ANSI},
+  year={1978}
+}
+
+@misc{HIPwebsite,
+  title = "HIP",
+  url = "https://rocmdocs.amd.com/en/latest/Programming_Guides/HIP-GUIDE.html",
+  year = "2020"
+}
+
+@article{Julia-lang,
+  title={Julia: A fresh approach to numerical computing},
+  author={Bezanson, Jeff and Edelman, Alan and Karpinski, Stefan and Shah, Viral B},
+  journal={SIAM {R}eview},
+  volume={59},
+  number={1},
+  pages={65--98},
+  year={2017},
+  publisher={SIAM},
+  doi={10.1137/141000671}
+}
+
+@misc{libceed-user-manual,
```

Comment:
Thanks @jeremylt for addressing some of the reviewer's comments. 
Should we update this to the v0.8, though? 
@jedbrown should that be updated on Zenodo, so that we refer to the most recent version?

Reply:
We'll be releasing 0.8.1 right before the JOSS acceptance, yes? So we could cite that version.